src/app/pages/OpenWalletPage/Features/ImportAccountsSelectionModal/index.tsx

```diff
@@ -141,10 +141,12 @@ export function ImportAccountsSelectionModal(props: ImportAccountsSelectionModal
               onClick={onPrev}
             />
             <Box>
-              {t('openWallet.importAccounts.pageNumber', 'Page {{ pageNum }} of {{ totalPages }}', {
-                pageNum: importAccounts.accountsSelectionPageNumber + 1,
-                totalPages: numberOfAccountPages,
-              })}
+              <span>
+                {t('openWallet.importAccounts.pageNumber', 'Page {{ pageNum }} of {{ totalPages }}', {
```

Comment:
> src/app/pages/OpenWalletPage/Features/ImportAccountsSelectionModal/index.tsx
L~144 can we wrap {t('openWallet.importAccounts.pageNumber' with span so pagination counter is working when translation is on

Added. Nice, it works. No idea why it works.